test-configs.yaml

```diff
@@ -1210,13 +1284,16 @@ test_configs:
     test_plans: [boot]
 
   - device_type: qemu_arm
-    test_plans: [boot_qemu, simple_qemu, v4l2_compliance_qemu_vivid]
+    test_plans: [boot_qemu, simple_qemu, v4l2_compliance_qemu_vivid, uefi-arm-gicv2, uefi-arm-gicv3]
```

Comment:
Might be worth using the expanded list syntax as this is starting to grow (easier to diff and maintain imho):
```yaml
    test_plans:
      - boot_qemu
      - simple_qemu
      - v4l2_compliance_qemu_vivid
      - uefi-arm-gicv2
      - uefi-arm-gicv3
```